README.md

```diff
@@ -31,3 +31,68 @@ There are capistrano tasks to connect to the Solr UI for managing solr that can
 
 * Would it be better to go from a server on Princeton's network (e.g., the bibdata-staging worker machine), so that Developer's internet connection doesn't affect benchmarking
 
+## Running from loadtest.lib.princeton.edu
+* Ensure the User Defined Variables in the Solr test plan are correct and saved for the environment
+* Copy the test file onto the remote host
+```bash
+scp solr_test_plan.jmx deploy@loadtest.lib.princeton.edu:~/
+```
+* Copy the keywords file onto the remote host
+```bash
+scp keywords.csv deploy@loadtest.lib.princeton.edu:~/
+```
+* SSH onto the box
+```bash
+ssh deploy@loadtest.lib.princeton.edu
+```
```

Comment:
Should we delete/move the existing `test_results` directory before running?  Or is it desired to dump all results into the same directory?

Reply:
Right now the box is configured so it *should* delete old results on each new test run. That's what [this config block in Princeton Ansible](https://github.com/pulibrary/princeton_ansible/blob/main/roles/loadtest/tasks/main.yml#L24-L28) should do. 

What do you think? I wasn't thinking of the server as a place for long-term storage, but that each time we run, we should save the artifacts we want to keep in other locations, but if it's easier to keep each "official" run as its own set of artifacts on the server that's ok with me, as long as we don't absolutely trust to them being there long term.

Reply:
+1 to not using the server for long-term storage of test artifacts.  The only thing I was wondering about was creating the HTML report at the end -- does it have the ability to make graphs across multiple runs, and if so, how do we set it up to do so?